wrangler-service/src/main/java/co/cask/wrangler/service/bigquery/BigQueryService.java

```diff
@@ -387,12 +387,12 @@ public void specification(HttpServiceRequest request, final HttpServiceResponder
             row.add(fieldName, LocalDate.parse(fieldValue.getStringValue()));
             break;
 
-          case DATETIME:
           case TIMESTAMP:
             long tsMicroValue = fieldValue.getTimestampValue();
             row.add(fieldName, getZonedDateTime(tsMicroValue));
             break;
 
+          case DATETIME:
```

Comment:
If it's always ISO 8601, should we convert to a timestamp automatically?

If somebody wants to write to a datetime column using the bigquery sink, how does it work?

Reply:
We can convert string to ZonedDateTime. However, `DATETIME` does not have timezone information. So we will have to assume that the value is in UTC or ask user to enter timezone information probably in bigquery connection config.

Another option is to keep the type of the value as is and let the user provide timezone information using Natural Date directive. 
![image](https://user-images.githubusercontent.com/14131070/50123166-7fe7a600-0214-11e9-8fed-5fcf45eb07fb.png)

In plugins, we do not have a notion of timestamp without time zone. Thats why sink only supports DATE, TIME and TIMESTAMP bigquery types.
https://github.com/data-integrations/google-cloud/blob/release/0.11/src/main/java/co/cask/gcp/bigquery/BigQuerySink.java#L189

Reply:
I see. The approach in this PR seems fine to me then